### DISCUSSION TITLE
Mod Support

### DISCUSSION THREAD

**Discussion by CygnSoft** [lloesche/valheim-server-docker#481] [Q&A]
Hello,

this is basically my first Docker Container as a Gameserver and I'm wondering if it is still possible to install several Mods from sources like Nexus Mods.

Usually I'm downloading the Mod and transfer it via FTP to my gameservers mod folder. Would it be the same in this case? Or is there any possibility to give the Mods Source to the environment variables?

Greetings:
Daniel

**Comment by lloesche** [comment]
There's BepInEx support so you can run any mod BepInEx supports by copying it into the corresponding directories. See [README.md](https://github.com/lloesche/valheim-server-docker#modding) for more information.
